docs/cli.rst

```diff
@@ -22,8 +22,9 @@ All the power of ``glom``, without even opening your text editor!
 
    --help / -h                     show this help message and exit
    --target-file TARGET_FILE       path to target data source (optional)
-   --target-format TARGET_FORMAT   format of the source data (json or python)
-                                     (defaults to 'json')
+   --target-format TARGET_FORMAT
+                                   format of the source data (json, python, toml
```

Comment:
(Even if for whatever reason we don't merge this, this was missing `yaml` in the `--help`)

Reply:
Good catch! I'm an oxford comma fan, so I'll probably just self-apply this suggestion:

```suggestion
                                   format of the source data (json, python, toml,
```